app/views/Partners/PartnersOverview.tsx

```diff
@@ -0,0 +1,127 @@
+} from 'react-native';
+import {
+  NavigationParams,
+  NavigationScreenProp,
+  NavigationState,
+} from 'react-navigation';
+
+import { Images, Icons } from '../../assets';
+import { NavigationBarWrapper } from '../../components/NavigationBarWrapper';
+import { Typography } from '../../components/Typography';
+import Colors from '../../constants/colors';
+
+import { SvgXml } from 'react-native-svg';
+import colors from '../../constants/colors';
+
+// For fixing image width issues
+const win = Dimensions.get('window');
+
+type PartnersScreenProps = {
+  navigation: NavigationScreenProp<NavigationState, NavigationParams>;
+};
+
+const PartnersScreen = ({ navigation }: PartnersScreenProps): JSX.Element => {
+  const { t } = useTranslation();
+  const [toggleActive, setToggleActive] = useState(false); // mocked
+  const navigateToViewHAs = () => navigation.navigate('PartnersEdit');
+
+  return (
+    <NavigationBarWrapper
+      title={t('authorities.title')}
+      includeBackButton={false}
+      onBackPress={() => {}}>
+      <ScrollView
+        style={styles.backgroundWrapper}
+        alwaysBounceVertical={false}
+        contentContainerStyle={{ flexGrow: 1 }}>
+        <Image
+          // Hard code to the dimensions of the image. Fixes ScrollView issues
+          style={{ width: win.width, height: (win.width * 900) / 1125 }}
+          resizeMode={'contain'}
+          source={Images.Doctors}
+        />
+        <View style={{ height: 20 }} />
+        <View style={styles.horizontalPadding}>
+          <Typography use={'headline2'}>{t('authorities.title')}</Typography>
+          <View style={{ height: 8 }} />
+          <Typography use={'body1'}>{t('authorities.info_body')}</Typography>
+        </View>
+        <View style={{ height: 20 }} />
+        {/* Outside horizontal padding so touchable is full width */}
+        <View style={styles.divider} />
+        <TouchableHighlight
+          underlayColor={Colors.UNDERLAY}
+          onPress={navigateToViewHAs}>
+          <View
+            style={[
+              styles.horizontalPadding,
+              {
+                flexDirection: 'row',
+                justifyContent: 'space-between',
+                paddingVertical: 20,
+              },
+            ]}>
+            <Typography use={'body1'} style={{ fontWeight: '500' }}>
+              {t('authorities.view_button_label')}
+            </Typography>
+            <SvgXml xml={Icons.ChevronRight} height={20} />
+          </View>
+        </TouchableHighlight>
+        <View style={[styles.divider, { marginHorizontal: 24 }]} />
+      </ScrollView>
+      <View style={styles.bottomSheet}>
+        <Typography
+          style={{
+            // Prevent from forcing overflow on parent
+            flex: 1,
+          }}>
+          {t('authorities.automatically_follow')}
+        </Typography>
+        <View style={{ width: 16 }} />
+        <Switch
+          trackColor={{ true: '#6979F8', false: '#6979F8' }}
```

Comment:
should we be using a common style here?

Reply:
Good call, I'll ensure these are in Colors